lib/lotus/routing/resource/action.rb

```diff
@@ -179,7 +179,7 @@ def rest_path
         # @api private
         # @since 0.1.0
         def as
-          namespace.relative_join(resource_name, self.class.named_route_separator).to_sym
+          namespace.relative_join(resource_name.to_s.gsub("/","_"), self.class.named_route_separator).to_sym
```

Comment:
@jodosha That is an amazing tidbit! I tried benchmarking, and using raw strings indeed takes 14% more time to run versus using frozen constants.